docs/admin-api/service.md

```diff
@@ -31,3 +31,22 @@
   - GET /?service
   - x-minio-operation: status
   - Response: On success 200, return json formatted StorageInfo object.
+
+### Lock Management APIs
+* ListLocks
+  - GET /?lock&bucket=mybucket&prefix=myprefix&older-than=rel_time
+  - x-minio-operation: list
+  - Response: On success 200, json encoded response containing all locks held, older than rel_time. e.g, older than 3 hours.
+  - Possible error responses
+    - ErrInvalidBucketName
```

Comment:
Can you add a simple paragraph somewhere to explain the generic format of the error ? (it must be something like what a S3 would return, right ?)